src/CloudKit/CKCompat.cs

```diff
@@ -30,6 +35,94 @@ public CKNotificationID ()
 		{
 		}
 	}
+
+	public partial class CKContainer {
+
+		[iOS (8, 0), Mac (10, 10)]
+		[Obsolete ("Use 'DiscoverAllIdentities' instead. NotSupportedException'")]
```

Comment:
too many different patterns in the PR

Use `"Always throw a 'NotSupportedException'. Use 'X' instead."`

i.e. start with the fact it won't work, then suggest alternative.

Starting with a suggestion might be discarded before reaching the not-working part, e.g. horizontal scrolling build warnings inside an IDE.